core/src/test/java/org/apache/iceberg/TestMetrics.java

```diff
@@ -613,6 +626,99 @@ public void testTruncateBinaryMetricsMode() throws IOException {
     assertBounds(1, Types.BinaryType.get(), expectedMinBounds, expectedMaxBounds, metrics);
   }
 
+  @Test
+  public void testSortedColumnMetrics() throws IOException {
+    File tableDir = temp.newFolder();
+    tableDir.delete(); // created by table create
+
+    SortOrder sortOrder = SortOrder.builderFor(SIMPLE_SCHEMA)
+        .asc("booleanCol")
+        .asc("intCol")
+        .asc("longCol")
+        .asc("decimalCol")
+        .asc("stringCol")
+        .asc("dateCol").build();
+    PartitionSpec spec = PartitionSpec.unpartitioned();
+    Table table = TestTables.create(tableDir, "test", SIMPLE_SCHEMA, spec, sortOrder, FORMAT_V2);
```

Comment:
Why does this test create a v2 table? Won't this work with v1 as well?

Reply:
Originally I wanted to not increase test runtime that much, as metrics seemed pretty independent of formatVersion.

Changed to make the tests parameterized to run with v1 and v2, or let me know if prefer the original.